pkg/convert/python/python.go

```diff
@@ -273,6 +273,9 @@ func (c *PythonContext) generateManifest(ctx context.Context, pack Package, vers
 	// Generate each field in the manifest
 	generated.GeneratedFromComment = pack.Info.ProjectURL
 	generated.Package = c.generatePackage(ctx, pack, version)
```

Comment:
You  need to modify generatePackage to set `Dependencies.ProviderPriority` and likewise in the subpackage blocks

Reply:
Ahh yeah, I should add test for that too and It looks like I missed adding tests for other additions as well.